frontend/src/core/cells/runs.ts

```diff
@@ -54,7 +54,16 @@ const {
     let run = state.runMap.get(runId);
     if (run) {
       runIds = state.runIds;
+      // Shallow copy the run to avoid mutating the existing run
+      run = { ...run };
     } else {
+      // If it is a brand new run and the cell code is "pure markdown",
+      // we don't want to show the trace since it's not helpful.
+      // This spams the tracing because we re-render pure markdown on keystrokes.
+      if (isPureMarkdown(code)) {
```

Comment:
skipping pure markdown since it spams the keystrokes. lmk your thoughts though @Light2Dark if you think we should keep it

Reply:
thanks, missed this 🙈